dashboard/app/controllers/scripts_controller.rb

```diff
@@ -100,19 +100,12 @@ def destroy
   end
 
   def edit
-    beta = params[:beta].present?
```

Comment:
I think removing this line will give you merge conflicts when you merge to the adhoc branch next: 
![Screen Shot 2020-12-04 at 3 22 53 PM](https://user-images.githubusercontent.com/8001765/101224808-a17e0a00-3644-11eb-883b-442ee72e0e8c.png)
You may have figured this out already, but what I would suggest is to let the code in this PR overwrite the temp code on the adhoc. then, when @bethanyaconnor is done updating the import script to set the is_imported flag, we should still be able to get the new script editor for any of the migrated scripts on the adhoc.